### ISSUE TITLE
Amiibos getting corrupted after being written to on splatoon

### ISSUE THREAD

**Issue by yozora1209** [Poohl/joycontrol#1]
I went to write to the amiibo in the home menu, it went just fine and my user was registered, then i went to splatoon, went to the amiibo box on the plaza, and selected for it to write the gear, camera, etc to the amiibo, it said the writing failed, but i remembered that was on purpose, so i happly went to the equip menu and went to scan the amiibo there, 1 minute passed, 2 minutes passed, 5 minutes passed and the splatoon 2 kept on the "reading amiibo do not mess with it", so i decided to close joycontrol and try to use the already written bin file with joycon droid, hoping it was an issue with joycontrol, but no, the file is corrupted, joycondroid doesnt read it neither

yes i know joycondroid cant write to the amiibo but im using the already written version for read, so the issue is with joycontrol itself corrupting the file

**Comment by Poohl** [comment]
Yes, I noticed that exact problem too and I'm not certain what causes it. I also had one binfile be completely messed up to the point where it wasn't even 540 bytes in length. I'm working on that, might take a while.

The latest version now quits out of the 'writing amiibo screen' when done, but saving gear to the amiibo in splatoon still appears to corrupt the dump. It seems only registering owners works any good although I don't understand the difference between both processes. Both write to the same memory segments.

**Comment by Poohl** [comment]
Just FYI: the current state is, it crashes and writes "no equipment saved" to the amiiboo...
Will work on this again as V12 seems to be fixed now.
